doc/examples/jest_react/package.json

```diff
@@ -13,7 +13,7 @@
 		"test": "jest"
 	},
 	"devDependencies": {
-		"axe-core": "^3.1.1",
+		"axe-core": "^3.2.2",
```

Comment:
Keeping this up-to-date seems tedious. Since we're already refactoring this a bit, would a `*` or `file://` be better?

Reply:
It seems since these are primarily intended to be examples to be consumed, neither `*` or `file:...` seem appropriate if we want to promote best practices.

Reply:
Sounds good.